packages/statemanager/README.md

```diff
@@ -13,7 +13,7 @@
 
 To obtain the latest version, simply require the project using `npm`:
 
-```shell
+```bash
```

Comment:
I can't follow the logic why this should be updated for `embedme` TBH, since these small install notes have totally nothing to do with this conversion.

My tendency would be revert this, this is only causing (small) harm by cluttering find-and-replace opportunities in the future, also `shell` is the more generic term I would think (since `bash` is just one certain shell flavor).

(eventually you can completely kill out the full commit to make it cleaner, by doing a `git rebase -i master` and then deleting out the commit and force push the branch)

Reply:
Fixed

Reply:
Ah, yes, `sh` should be good, but then do it monorepo-wide with a search-and-replace.

Reply:
I realized since we're not embedding any of the shell examples, it isn't entirely necessary to change shell annotations to an annotation that embedme understands.